src/site/_filters/language-list.js

```diff
@@ -15,52 +15,55 @@
  */
 
 const {
-  defaultLanguage,
   languageNames,
+  defaultLanguage,
   languageOrdering,
-} = require('../../../lib/utils/language');
+} = require('../../lib/utils/language');
+
 const {
   getDefaultUrl,
-  getLocaleSpecificUrl,
   getTranslatedUrls,
-} = require('../../_filters/urls');
-const {i18n} = require('../../_filters/i18n');
+  getLocaleSpecificUrl,
+} = require('../_filters/urls');
+
+/**
+ * Look up all supported languages for a specific post
+ *
+ * @param {String} lang The current lang of a post to look up.
+ * @param {EleventyCollectionItem} post The post to look up.
+ * @return {String[][]} The list of supported languages for a specific post
```

Comment:
Question: is that a valid type?

Reply:
Yes, this function will return the array of supported languages like:

```
[
  [ 'en', '/i18n/en/webrtc-standard-announcement/', 'English' ],
  [ 'es', '/i18n/es/webrtc-standard-announcement/', 'Español' ],
  [ 'pt', '/i18n/pt/webrtc-standard-announcement/', 'Português' ],
  [ 'ru', '/i18n/ru/webrtc-standard-announcement/', 'Русский' ],
  [ 'zh', '/i18n/zh/webrtc-standard-announcement/', '中文' ],
  [ 'ja', '/i18n/ja/webrtc-standard-announcement/', '日本語' ],
  [ 'ko', '/i18n/ko/webrtc-standard-announcement/', '한국어' ]
]
```